src/tsconfig.base.json

```diff
@@ -0,0 +1,41 @@
+{
+  "extends": "../tsconfig.json",
+
+  "compilerOptions": {
+    "checkJs": true,
+    "downlevelIteration": true,
+    "target": "es5",
+    "strictNullChecks": true,
+    "noImplicitAny": true,
+    "lib": ["dom", "esnext", "es6"],
+
+    "paths": {
+      "#3p/*": ["./3p/*"],
+      "#ads/*": ["./ads/*"],
+      "#build/*": ["./build/*"],
+
+      "#builtins/*": ["./src/builtins/*"],
+      "#compiler": ["./src/compiler"],
+      "#compiler/*": ["./src/compiler/*"],
+      "#core/*": ["./src/core/*"],
+      "#experiments": ["./src/experiments"],
+      "#experiments/*": ["./src/experiments/*"],
+      "#inabox/*": ["./src/inabox/*"],
+      "#polyfills": ["./src/polyfills"],
+      "#polyfills/*": ["./src/polyfills/*"],
+      "#preact": ["./src/preact"],
+      "#preact/*": ["./src/preact/*"],
+      "#purifier": ["./src/purifier"],
+      "#purifier/*": ["./src/purifier/*"],
+      "#service": ["./src/service"],
+      "#service/*": ["./src/service/*"],
+      "#utils": ["./src/utils"],
+      "#utils/*": ["./src/utils/*"],
+
+      "#test/*": ["./test/*"],
+      "#testing/*": ["./testing/*"],
+    }
+  },
+
+  "include": [],
```

Comment:
is this necessary?

Reply:
The root `include` includes a bunch of stuff for VSCode/Intellisense sake, this overwrites so anything that extends it has a "blank slate". Though, since everything that extends it should define their own `include`s anyway, I guess this would be fine to remove